components/prize_pool/commons/prize_pool_legacy.lua

```diff
@@ -171,14 +170,24 @@ function LegacyPrizePool.mapSlot(slot, mergeSlots)
 	local opponentsInSlot = #slot
 	Table.iter.forEachPair(CACHED_DATA.inputToId, function(parameter, newParameter)
 		local input = slot[parameter]
-		if newParameter == 'seed' then
+		if input == 'q' and slot.link then
+			-- Use qualifier display
+			if not slot.link:find('^%[%[') then
+				slot.link = '[[' .. slot.link .. ']]'
+			end
+			LegacyPrizePool.handleSeed(newData, slot.link, opponentsInSlot)
+		elseif input == 'q' and slot.link2 then
+			-- Use qualifier display
+			if not slot.link2:find('^%[%[') then
+				slot.link2 = '[[' .. slot.link2 .. ']]'
+			end
+			LegacyPrizePool.handleSeed(newData, slot.link2, opponentsInSlot)
```

Comment:
Why's link2 needed?

Reply:
The last [test page](https://liquipedia.net/leagueoflegends/LPL/2018/Spring) uses `points2` in combination with `link2`. Since the qualifier is identified by link, it is not necessary to actually match with points2 if there is a `q` input. If you want, i can add a match on the parameter index for this.

Reply:
Looking at the count, I'm leaning on swapping manually. There are 22 on League, 3 on dota2 and 6 on Brawl Stars across all wikis

Reply:
Pages to check further:

- https://liquipedia.net/leagueoflegends/BRCC/2020/Summer/Access (no link set, column vanishes)
- https://liquipedia.net/leagueoflegends/Intel_Extreme_Masters/Season_VI/Cologne (and siblings) (numeric values and q for one column)

Reply:
```suggestion
```